src/main/java/mate/academy/spring/dto/request/MovieRequestDto.java

```diff
@@ -1,9 +1,10 @@
 package mate.academy.spring.dto.request;
 
+import jakarta.validation.constraints.Min;
 import jakarta.validation.constraints.NotNull;
 
 public class MovieRequestDto {
-    @NotNull
+    @Min(value = 2)
```

Comment:
[`@NotBlank`](https://www.baeldung.com/java-bean-validation-not-null-empty-blank)